WPMedia/engine-theme-sdk
# tmedia 842: fix: Ensure no error out if no sites key

-  a non-multisite client might not have a sites object.  https://arcpublishing.atlassian.net/jira/software/c/projects/TMEDIA/boards/875?modal=detail&selectedIssue=TMEDIA-842&assignee=5e387d6a1b1d910e5dfd7a9b

# Testing
- add a blocks.json of below to the src folder 
- run `rm -rf node_modules && npm i`. should only install new york timezone. 
- Should not see fallback timezones used on error with no sites 

```
howardj@arc-20750 engine-theme-sdk % ls node_modules/@wpmedia/timezone/       
America         EET.js          HST.js          MST7MDT.js      en_US.js        package.json
CET.js          EST.js          MET.js          PST8PDT.js      index.js        rfc822.js
CST6CDT.js      EST5EDT.js      MST.js          WET.js          locales.js      zones.js
howardj@arc-20750 engine-theme-sdk % ls node_modules/@wpmedia/timezone/America 
New_York.js     index.js
```


```
{
	"localeList": [
		"en",
		"sv",
		"no",
		"fr",
		"de",
		"es",
		"ja",
		"ko",
		"pt"
	],
	"values": {
		"default": {
			"siteProperties": {
				"dateLocalization": {
					"language": "en",
					"timeZone": "America/New_York",
					"dateTimeFormat": "%B %d, %Y at %l:%M %P %Z",
					"dateFormat": "%B %d, %Y"
				},
				"nativoIntegration": "false",
				"navBarBackground": "none",
				"breakpoints": {
					"small": 0,
					"medium": 768,
					"large": 992
				},
				"imageWidths": [
					84,
					105,
					158,
					274,
					377,
					400,
					600,
					768,
					800,
					1024,
					1440,
					1600
				],
				"aspectRatios": [
					"3:2",
					"4:3",
					"1:0",
					"16:9",
					"1:1"
				],
				"fontUrl": [
					"https://fonts.googleapis.com/css?family=Abel"
				]
			},
			"styleProperties": {
				"primary-font-family": "'Abel', Sans-Serif",
				"secondary-font-family": "'Abel', Sans-Serif",
				"primary-color": "#fff"
			}
		}
	}
}

``
